assets/scss/base/typography/_headings.scss

```diff
@@ -83,6 +83,23 @@ Styleguide Typography.Heading Sizes
   @include core-24();
 }
 
+
+/*
+Bold h3 heading
+
+Experimental: Bold h3 heading for tcs-payments
+
+Markup:
+<h3 class="{{modifier_class}}">header example</h3>
+
+.heading-24-bold - a bold font size 24px header
+*/
+
+.heading-24-bold {
```

Comment:
Please use styles that already exist: `.h2-heading`

https://github.com/hmrc/assets-frontend/blob/master/assets/scss/base/typography/_headings.scss#L37